dbms/src/Storages/Page/V3/LogFile/LogWriter.h

```diff
@@ -64,11 +64,11 @@ namespace PS::V3
  * Recyclable record format:
  *
  * +--------------+-----------+-----------+----------------+--- ... ---+
- * |CheckSum (8B) | Size (2B) | Type (1B) | Log number (4B)| Payload   |
+ * |CheckSum (8B) | Size (2B) | Type (1B) | Log number (8B)| Payload   |
  * +--------------+-----------+-----------+----------------+--- ... ---+
  *
  * Same as above, with the addition of
- * Log number = 32bit log file number, so that we can distinguish between
+ * Log number = 64bit log file number, so that we can distinguish between
```

Comment:
Is it compatible with old data?

Reply:
Old data is 64 bit. This is just a change of code comment